### DISCUSSION TITLE
How do I listen for Ctrl+Enter Command on Editor ?

### DISCUSSION THREAD

**Discussion by saumya66** [facebook/lexical#3432] [Q&A] [answered]
I'm trying to trigger a function when the user presses Ctrl+Enter while focused on the editor. I found .registerCommand() but it does not have a command to detect ctrl+enter.

would love to know it's possible to do so now & if so how?

**Comment by thegreatercurve** [comment, marked as answer]
Use the `KEY_ENTER_COMMAND` command, and the payload contains the DOM event to check if the Ctrl key is also being pressed as well.

**Reply by saumya66** [reply]
thanks @thegreatercurve it works!

**Comment by saumya66** [comment]
Is there any way that after detecting the Enter keypress using KEY_ENTER_COMMAND, I can avoid its default behavior i.e cursor going to the next line? 

Trying to do this as the cursor going down, changes the states once.

**Reply by thegreatercurve** [reply]
Use `event.preventDefault` and `return true` from the command to prevent is calling the next default command in the array.
